src/gui/generalsettings.cpp

```diff
@@ -359,6 +359,10 @@ void GeneralSettings::slotUpdateChannelChanged()
             return tr("daily");
         }
 
+        if (channel == QStringLiteral("enterprise")) {
+            return tr("enterprise");
+        }
+
```

Comment:
Can't we just replace these ifs by returning the string wrapped in a `tr`?

Reply:
for `tr`, you need the string to be a constant such that it is seen by the extractor and translators can translate them